docs/unified-test-documentation/dasharo-compatibility/31U-heads-bootloader-support.md

```diff
@@ -63,3 +63,319 @@ bootloader.
+
+This test aims to verify that the `Add GPG key to running BIOS and reflash`
+option in the `GPG Management Menu` is available and works correctly.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+1. Additional `USB storage` - at least 1GB, with GPG public key
+
+**Test setup**
+
+1. Proceed with the
+    [Test cases common documentation](#test-cases-common-documentation) section.
+
+**Test steps**
+
+1. Plug the `USB storage` into DUT.
+1. Power on the DUT.
+1. Wait for the `Default boot menu` appears.
+1. Select the `Continue to the main menu` option using the arrow keys and Enter.
+1. Select the `Options -->` option in the `Heads boot menu`.
+1. Select the `GPG Options -->` option in the `HEADS Options` submenu.
+1. Select the `Add GPG key to running BIOS and reflash` option in the
+   `GPG Management Menu`.
+1. Choose `<Yes>` in the displayed `GPG public key required` window using the
+   arrow keys and Enter.
+1. Choose GPG public key from the `USB storage` and press Enter.
+1. Select the `Options -->` option in the `Heads boot menu`.
+1. Select the `GPG Options -->` option in the `HEADS Options` submenu.
+1. Select the `List GPG keys in your keyring` option in the
+   `GPG Management Menu`.
+1. Note the results.
+
+**Expected result**
+
+The `GPG Keyring` window should contain information about the given GPG key.
+
+## HDS012.001 Add GPG key to standalone BIOS image and flash
+
+**Test description**
+
+This test aims to verify that the
+`Add GPG key to standalone BIOS image and flash` option in the
+`GPG Management Menu` is available and works correctly.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+1. Additional `USB storage` - at least 1GB, with GPG public key and BIOS image
+
+**Test setup**
+
+1. Proceed with the
+    [Test cases common documentation](#test-cases-common-documentation) section.
+
+**Test steps**
+
+1. Plug the `USB storage` into DUT.
+1. Power on the DUT.
+1. Wait for the `Default boot menu` appears.
+1. Select the `Continue to the main menu` option using the arrow keys and Enter.
+1. Select the `Options -->` option in the `Heads boot menu`.
+1. Select the `GPG Options -->` option in the `HEADS Options` submenu.
+1. Select the `Add GPG key to standalone BIOS image and flash` option in the
+   `GPG Management Menu`.
+1. Choose `<Yes>` in the displayed `GPG public key required` window using the
+   arrow keys and Enter.
+1. Choose GPG public key from the `USB storage` and press Enter.
+1. Choose BIOS image(*.rom) from the `USB storage` and press Enter.
+1. Choose `<Yes>` in the displayed `Flash ROM?` window using the arrow keys and
+   Enter.
+1. Select the `Options -->` option in the `Heads boot menu`.
+1. Select the `GPG Options -->` option in the `HEADS Options` submenu.
+1. Select the `List GPG keys in your keyring` option in the
+   `GPG Management Menu`.
+1. Note the results.
+
+**Expected result**
+
+The `GPG Keyring` window should contain information about the given GPG key.
```

Comment:
I think, that in this place we should add some `Expected result`.

Reply:
Added